Package.swift

```diff
@@ -17,21 +17,23 @@ let package = Package(
     )
   ],
   dependencies: [
-    .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "0.1.3"),
     .package(url: "https://github.com/ReactiveCocoa/ReactiveSwift", from: "6.4.0"),
+    .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "0.1.3"),
+    .package(url: "https://github.com/pointfreeco/xctest-dynamic-overlay", from: "0.1.0"),
   ],
   targets: [
     .target(
       name: "ComposableArchitecture",
       dependencies: [
-        .product(name: "CasePaths", package: "swift-case-paths"),
         "ReactiveSwift",
+        .product(name: "CasePaths", package: "swift-case-paths"),
+        .product(name: "XCTestDynamicOverlay", package: "xctest-dynamic-overlay"),
```

Comment:
@stephencelis I noticed that you are just using `"CasePaths"` and `"XCTestDynamicOverlay"` directly without needing an explicit `.product` entry. When I tried that Xcode throws an error telling me I have to use this explicit form. It's not an issue, but I was just curious if you had any idea why I can't use just the product name by itself?